diff --git a/node_modules/metro-config/src/loadConfig.js b/node_modules/metro-config/src/loadConfig.js
index 7ac9d88..194c3b3 100644
--- a/node_modules/metro-config/src/loadConfig.js
+++ b/node_modules/metro-config/src/loadConfig.js
@@ -1,4 +1,4 @@
-"use strict";
+﻿"use strict";
 
 Object.defineProperty(exports, "__esModule", {
   value: true,
@@ -289,7 +289,7 @@ async function loadConfigFile(absolutePath) {
       }
     } catch (e) {
       try {
-        const configModule = await import(absolutePath);
+        const configModule = await import(require("url").pathToFileURL(absolutePath).href);
         config = await configModule.default;
       } catch (error) {
         let prefix = `Error loading Metro config at: ${absolutePath}\n`;
@@ -345,3 +345,4 @@ function searchForConfigFile(absoluteStartDir, absoluteStopDir) {
   }
   return null;
 }
+
